### DISCUSSION TITLE
base64 image data preview

### DISCUSSION THREAD

**Discussion by hulinux** [directus/directus#13498] [Feature Requests]
The image data is stored in base64 encoding. Can the display function be added to the file preview component, and the PDF format file can also be added to the display function?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
